cypress/e2e/models/administration/jira/jira.ts

```diff
@@ -0,0 +1,236 @@
+        this.selectType();
+        this.selectCredentials();
+        this.configureInsecure();
+        if (!toBeCanceled) {
+            submitForm();
+            this.validateState();
+        } else {
+            cancelForm();
+            notExists(this.name);
+        }
+    }
+
+    /**
+     * This method edits existing Jira connection according to object values
+     *
+     * @param jiraConnectionData brings new values that should be applied to instance.
+     * @param toBeCanceled is responsible for canceling editing instead of submitting if set to `true`
+     */
+    public edit(jiraConnectionData: JiraConnectionData, toBeCanceled = false): void {
+        Jira.openList();
+        performRowAction(this.name, editAction);
+        const oldValues = this.storeOldValues();
+        this.init(jiraConnectionData);
+        this.fillName();
+        this.fillUrl();
+        this.selectType();
+        this.selectCredentials();
+        this.configureInsecure();
+        if (!toBeCanceled) {
+            // Edit action is confirmed, submitting form
+            submitForm();
+        } else {
+            // Edit action was canceled
+            this.init(oldValues);
+            cancelForm();
+        }
+        this.validateState();
+        exists(this.name);
+    }
+
+    /**
+     * This method deletes Jira connection
+     *
+     * @param toBeCanceled is responsible for canceling deletion instead of submitting if set to `true`
+     */
+    public delete(toBeCanceled = false): void {
+        Jira.openList();
+        performRowAction(this.name, deleteAction);
+        if (toBeCanceled) {
+            click(cancelButton);
+            exists(this.name);
+        } else {
+            click(confirmButton);
+            notExists(this.name);
+        }
+    }
+
+    /**
+     * This method validates all fields values of Jira connection after creation
+     */
+    private validateState(): void {
+        cy.get(tdTag, { timeout: 120 * SEC })
+            .contains(this.name, { timeout: 120 * SEC })
+            .closest(trTag)
+            .within(() => {
+                this.validateSingleState(jiraLabels.name, this.name);
+                this.validateSingleState(jiraLabels.url, this.url);
+                // Commenting lines below because of the bug that respective columns have same labels (MTA-766)
+                // this.validateSingleState(jiraLabels.type, this.type);
+                // this.validateSingleState(jiraLabels.connection, "Connected");
+            });
+    }
+
+    // This method is actually doing the same what utils/checkSuccessAlert does.
+    // TODO: To consider re-use of method in utils
+    /**
+     * This method is validating field of Jira connection
+     *
+     * @param FieldId is selector allowing to identify field to be validated
+     * @param text contains value to compare with field content
+     */
+    private validateSingleState(FieldId: string, text: string): void {
```

Comment:
Please rename this to validateTextPresence()

Reply:
Done